Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Merchant/TransporterConfig.hs

```diff
@@ -13,20 +13,20 @@
 -}
 {-# LANGUAGE TypeApplications #-}
 
-module Storage.Queries.TransporterConfig
+module Storage.Queries.Merchant.TransporterConfig
   {-# WARNING
     "This module contains direct calls to the table. \
   \ But most likely you need a version from CachedQueries with caching results feature."
     #-}
 where
 
 import Domain.Types.Merchant
-import Domain.Types.TransporterConfig
+import Domain.Types.Merchant.Configs.TransporterConfig
```

Comment:
`Domain.Types.Merchant.Configs.TransporterConfig` but `Storage.Tabular.Merchant.TransporterConfig`. Use same template to place things for everything after `.Merchant`